content/en/blog/_posts/2023-04-02-how-to-speed-up-pod-startup-from-kubelet-side.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: blog
+title: "How to speed up pod startup from kubelet side? "
+date: 2023-04-02T16:00:00+0000
+slug: speed-up-pod-startup
+---
+
+**Author**: Paco Xu(DaoCloud), Sergey Kanzhelev, Ruiwen Zhao(Google)
+
+How can pod start-up time be accelerated on nodes in large clusters? This is a common issue that cluster administrators may face.
```

Comment:
nit: this issue is not specific to *large* clusters, right? Even for an one-node cluster, we can use the techniques in this blog to speed up pod startups.

Reply:
uh I see. The QPS setting is mostly for large clusters.

Reply:
Most likely, you will meet the pod start-up thing in a large node or a large cluster.